stable/appmesh-controller/crds/crds.yaml

```diff
@@ -0,0 +1,382 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: meshes.appmesh.k8s.aws
```

Comment:
It would be nice if there was a way to not have to define these both in the controller package and the helm chart, but I feel like we should keep them in both places for now, for non helm users.

Reply:
The CRDs in the controller repo should be for development purpose only. The App Mesh users should use the CRDs from this repo as it doesn't require Helm to install them.